docs/04_web/02_web-components/02_grids/01_grid-pro/2-connected-data.mdx

```diff
@@ -101,3 +101,29 @@ This example creates a grid with a polling frequency of 10 seconds. Remember tha
 ...
 ```
 
+This example creates a grid with a server side row model allowing requesting data on demand as user scrolls.
```

Comment:
suggestion: we might need to tweak a bit further scenarios where we mention "row model". We don't do that right now for "client-side" + we'd need to also replace with the "better named" `grid-pro-client-side-datasource` component so we're super clear which one does X/Y.

Not a blocker for this one. Also, @nick-genesis might be able to help after the updated README around these two components.